.mlc-config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I'll suggest to match naming with: https://github.com/dotnet/iot/blob/main/.markdownlint.json

so something like `.markdown-link-checker.json`